docs/install/knative-with-any-k8s.md

```diff
@@ -559,7 +559,7 @@ The following command installs the GCP Pub/Sub Source:
 
    ```bash
    # This installs both the Source and the Channel.
-   kubectl apply --filename {{< artifact org="google" repo="knative-gcp" file="cloud-run-events.yaml" >}}
+   kubectl apply --filename {{< artifact repo="knative-gcp" file="cloud-run-events.yaml" >}}
```

Comment:
@nachocano these are only the right links because Google is publishing release artifacts for the Google org in the Knative bucket.  When we release the docs the link will be:
https://github.com/knative/knative-gcp/releases/tag/v0.12.1
Where you want:
https://github.com/google/knative-gcp/releases/tag/v0.12.1

Reply:
This is where the shortcode lives, if you want to customize how the artifact URL is generated for knative-gcp:
https://github.com/knative/website/blob/aa52fefa3f67fb01fc7e0779a97ee8d5b89fcde1/layouts/shortcodes/artifact.html#L36

Reply:
Thanks for the clarification Matt! Will add the org back. And will check out here if we can publish the nightly's in google's bucket.